site/src/sphinx/advanced-spring-webflux-integration.rst

```diff
@@ -0,0 +1,95 @@
+.. _advanced-spring-webflux-integration:
+
+Using Armeria with Spring WebFlux
+=================================
+
+.. note::
+
+    Visit `armeria-examples <https://github.com/line/armeria-examples>`_ to find the fully working example code.
+
+Spring framework provides powerful features which are necessary for building a web application, such as
+dependency injection, data binding, AOP, transaction, etc. In addition, if your Spring application integrates
+with Armeria, you can leverage the following:
+
+- rich support for Apache `Thrift <https://thrift.apache.org/>`_ and `gRPC <https://grpc.io/>`_,
+  including the documentation service
```

Comment:
How about:

- including [the fancy web console](https://github.com/line/armeria/blob/master/site/src/sphinx/server-docservice.rst) that enables you to send Thrift and gRPC requests from web browser.